frontend/src/components/newTrip/NewTripForm/NewTripForm.tsx

```diff
@@ -0,0 +1,41 @@
+import { PATH } from '@constants/path';
+import { Button } from 'hang-log-design-system';
+import type { FormEvent } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useNewTripMutation } from '@hooks/api/useNewTripMutation';
+import { useNewTripForm } from '@hooks/newTrip/useNewTripForm';
+
+import CitySearchBar from '@components/common/CitySearchBar/CitySearchBar';
+import DateInput from '@components/common/DateInput/DateInput';
+import { formStyling } from '@components/newTrip/NewTripForm/NewTripForm.style';
+
+const NewTripForm = () => {
+  const { newTripData, setCityData, setDateData, isAllInputFilled } = useNewTripForm();
+  const { mutate } = useNewTripMutation();
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    mutate(newTripData, {
+      onSuccess: moveToTripEditPageWithId,
+    });
+  };
+
+  const moveToTripEditPageWithId = (id: string) => {
+    navigate(PATH.EDIT_TRIP.replace(':tripId', id));
+  };
+
+  return (
+    <form css={formStyling} onSubmit={handleSubmit}>
+      <CitySearchBar setCityData={setCityData} />
+      <DateInput setDateData={setDateData} />
+      <Button variant="primary" disabled={!isAllInputFilled} style={{ width: '400px' }}>
```

Comment:
style을 사용하면 css 가 inline으로 박히지 않나요? 저희 emotion을 사용하는 만큼 inline css를 사용해요!
```suggestion
      <Button variant="primary" disabled={!isAllInputFilled} css={{ width: '400px' }}>
```

Reply:
애슐리 코드 보고 form쪽에서 ` > button`으로 주는 걸로 바꿨어요~~